src/frontend/src/components/public/PPQConnect/messages.ts

```diff
@@ -0,0 +1,27 @@
+export default {
+  Headings: {
+    Title: {
+      en: `Thank you for answering the PPQ`,
+    },
+    Subtitle: {
+      en: `Your answers will help us streamline the review of this PIA.`,
+    },
+    One: {
+      en: `1. Download your results`,
+    },
+    Two: {
+      en: `2. Connect with PCT`,
+    },
+    Three: {
+      en: `3. Wait`,
+    },
+  },
+  Text: {
```

Comment:
I would label this as step 2 or something more descriptive than text. Every value in here is text.